scripts/generateCLIDocs.ts

```diff
@@ -0,0 +1,86 @@
+import { writeFile } from "fs";
```

Comment:
I think it would make more sense to have this file is the lodestar-cli package. You could require the src instead of requiring `@chainsafe/lodestar-cli`

Reply:
done!